packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -0,0 +1,50 @@
+import type { ThemeTokens } from '../../../tokens/theme/theme.d';
+import type { TypographyPlatforms } from '../../../tokens/global/typography';
+
+export type ButtonMinHeight = '48px' | '40px' | '32px' | '28px';
+
+export type ButtonTypography = {
+  [key in TypographyPlatforms]: {
+    fonts: {
+      size: Record<string, keyof ThemeTokens['typography'][key]['fonts']['size']>;
+    };
+    lineHeights: Record<string, keyof ThemeTokens['typography'][key]['lineHeights']>;
+  };
+};
+
+const typography: ButtonTypography = {
+  onDesktop: {
+    fonts: {
+      size: {
+        large: 200,
```

Comment:
was wondering what are we getting from using the key here? can't we directly use the token here? `typography.onDesktop.fonts.size[200]`

Reply:
We want this to get computed from a theme, right? We use a key here which the component will then use to access the correct font size in its theme.